gallery/src/components/NxButton/NxButtonIconOnlyExample.tsx

```diff
@@ -11,9 +11,12 @@ import { NxButton, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxButtonIconOnlyExample = () =>
   <div className="nx-btn-bar">
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faExclamationTriangle}/></NxButton>
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faEdit}/></NxButton>
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faSave}/></NxButton>
+    <NxButton variant="icon-only"><NxFontAwesomeIcon aria-hidden={false}
+                                                     aria-label='alert'
```

Comment:
So when setting the label, we still need to set the hidden prop false?

Reply:
I think when there is a label aria-hidden shouldn't be set either way.

Reply:
> I think when there is a label aria-hidden shouldn't be set either way.

The problem is that fontawesome sets `aria-hidden` to true by default, so we need to override it to false.

Reply:
28b0cb95fae285bf202cb1c8c8cb423bc5a39f53 using undefined prevents the property from being set instead of being added with a value of false.

Reply:
> 28b0cb9 using undefined prevents the property from being set instead of being added with a value of false.

That commit is about `aria-disabled`; this thread is about `aria-hidden`...

Reply:
@rpokorny true, to clarify, the aria-hidden needs to be set because by default FontAwesomeIcon components will have aria-hidden as true, but now that I changed the example and moved the label to the button itself this is not a problem.